Gemfile

```diff
@@ -85,7 +85,9 @@ group :development, :test do
   gem "launchy"
 
   # Testing framework
+  gem "rspec"
```

Comment:
Do we need this. `rspec-rails` includes it I think.

Reply:
This is what's causing the issues when you try to run the tests locally. It needs to be bundled as the rails one brings in some additional extras which require more dependencies to be available.

Reply:
You're right, i was able to get it working using `rspec-rails`